notebook/gateway/managers.py

```diff
@@ -91,6 +91,16 @@ def _kernels_endpoint_default(self):
     def _kernelspecs_endpoint_default(self):
         return os.environ.get(self.kernelspecs_endpoint_env, self.kernelspecs_endpoint_default_value)
 
+    kernelspecs_resource_endpoint_default_value = '/kernelspecs'
```

Comment:
Does every endpoint need to be separately configurable? It seems to me like a single base path should be everything that's needed.

Reply:
I believe these are configurable (in NB2KG is done via just envs) for cases when the gateway server (JKG or JEG) is behind a reverse proxy.  This should be the last of them. (yeah, famous last words, I know. :smile:)

Reply:
ok, let's switch this over to a single configurable prefix before adding any more after this one.